Update Azure secrets docs: (#1730)
hashicorp/web-unified-docs: content/vault/v2.x (rc)/content/api-docs/secret/azure.mdx

@include 'alerts/enterprise-only.mdx'

Use this endpoint to import existing Azure credentials when creating a static role.
The import endpoint is only used during role creation. Any subsequent updates to the role
are applied using the standard Create/Update static role endpoint.

| Method | Path                               |
| :----- | :----------------------------------|
| `POST` | `/azure/static-roles/:name/import` |

### Path parameters

- `name` `(string: <required>)` – Name of the static role you want to
create or update.

### Request parameters

- `application_object_id` (`string: <required>`) -  Application object ID for
an existing service principal that is managed by the static role.
- `ttl` (`string: ""`) - Timespan of 1 month (`720h`) or more during
which the role credentials are valid. You can specify timespans
as time suffixed strings (`"1h"`) or an integer number of
seconds. Leave `ttl` unset or set to `0` to use the default value
of 2 years (`17520h`).
- `metadata` (`map[string]string: {}`) - A map of key/value pairs associated with the static role.
You can read metadata values from both the `/azure/static-roles/:name` and `/azure/static-creds/:name`

> endpoints. When credentials are fetched, role metadata is merged with any metadata provided in the request,

Reviewer: ```suggestion
endpoints. When Vault fetches credentials it merges role metadata with any metadata provided in the request,
```
Style correction: write in active voice
